### PR TITLE
Feedback for Ingress controller PR

### PR DESCRIPTION
**Pull request by axbarsan**

### CODE REVIEW

File: src/stores/appcatalog/actions.ts [hunk 1]
```diff
@@ -85,7 +83,7 @@ export const prepareIngressTabData = createAsynchronousAction<
 
       await Promise.all([
         dispatch(loadClusterApps({ clusterId: payload.clusterId })),
-        catalogLoadIndex(gsCatalog)(dispatch, () => state),
+        dispatch(catalogLoadIndex(gsCatalog)),
```
Comment (axbarsan) [inline src/stores/appcatalog/actions.ts:86]: Any reason for not using `dispatch` here?

File: src/components/Cluster/ClusterDetail/Ingress/__tests__/InstallIngressButton.ts [hunk 2]
```diff
@@ -28,8 +28,11 @@ const icApp = {
 };
 
 describe('InstallIngressButton', () => {
-  it.skip('renders without crashing', () => {
-    renderWithStore(InstallIngressButton, { cluster: defaultCluster });
+  it('renders without crashing', async () => {
+    // eslint-disable-next-line @typescript-eslint/require-await
+    await act(async () => {
+      renderWithStore(InstallIngressButton, { cluster: defaultCluster });
+    });
```
Comment (axbarsan) [inline src/components/Cluster/ClusterDetail/Ingress/__tests__/InstallIngressButton.ts:35]: magic?

Comment (ubergesundheit) [reply src/components/Cluster/ClusterDetail/Ingress/__tests__/InstallIngressButton.ts:35]: Seems I forgot to reenable this test in #1713

Comment (axbarsan) [reply src/components/Cluster/ClusterDetail/Ingress/__tests__/InstallIngressButton.ts:35]: The suggested fix is just to get rid of the `... not wrapped in 'act() ...'` error

File: src/stores/appcatalog/actions.ts (hunk 1 above)
Comment (ubergesundheit) [reply src/stores/appcatalog/actions.ts:86]: Very good question.. I copied this part from the initial implementation I did in the component. :shrug:

**Comment by ubergesundheit** [comment]
Thank you very much! Are you fine with me merging this into my other PR?

**Comment by axbarsan** [comment]
> Thank you very much! Are you fine with me merging this into my other PR?

Let's do it!

**Pull request merged**